### DISCUSSION TITLE
blake3 instead of sha256?

### DISCUSSION THREAD

**Discussion by hiqsociety** [ipfs/ipfs#490] [Polls]
blake3 instead of sha256?

**Comment by lbguilherme** [comment]
I guest any change in this regard should be based on technical arguments and not on popular vote?

I have questions: Why blake3 instead of sha256? And how such change can be made backward compatible with the existing use of sha256?

**Comment by 2color** [comment]
Kubo added support for Blake3: https://github.com/ipfs/kubo/pull/9187

Moreover, the [new direction Iroh](https://youtu.be/-ztYVfOzC_c?t=1277) is taking is also using Blake3.
